webapi/formatting.go

```diff
@@ -24,5 +24,5 @@ func blockURL(blockExplorerURL string) func(int64) string {
 }
 
 func dateTime(t int64) string {
-	return time.Unix(t, 0).Format("2 Jan 2006 15:04:05")
+	return time.Unix(t, 0).Format("2 Jan 2006 15:04:05 MST")
```

Comment:
Isn't UTC preferable here?

Reply:
Just as writing 2006 is a template value to be replaced with the actual year, MST is a template value to be replaced with the local timezone.

On my server, UTC is the timezone, so UTC will be written. Servers running on other timezones will have different values.